stock_custom_exchange_rate_date/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Customized Rate Date in Stock",
+    "version": "12.0.1.0.0",
+    "author": "Vauxoo",
+    "category": "stock",
+    "website": "http://www.vauxoo.com/",
+    "license": "LGPL-3",
+    "depends": [
+        "stock",
+        "purchase",
```

Comment:
purchase_stock only is the one needed to depend from.

Reply:
Done